frontend/public/module/k8s/cluster-settings.ts

```diff
@@ -87,13 +87,15 @@ export const getClusterVersionCondition = (
   return _.find(conditions, { type });
 };
 
-export const isProgressing = (cv: ClusterVersionKind): boolean => {
-  return !_.isEmpty(
-    getClusterVersionCondition(
-      cv,
-      ClusterVersionConditionType.Progressing,
-      K8sResourceConditionStatus.True,
-    ),
+export const updateInProgress = (cv: ClusterVersionKind): boolean => {
+  return (
+    !_.isEmpty(
+      getClusterVersionCondition(
+        cv,
+        ClusterVersionConditionType.Progressing,
+        K8sResourceConditionStatus.True,
+      ),
+    ) && getDesiredClusterVersion(cv) !== getLastCompletedUpdate(cv)
```

Comment:
Sorry, I probably wasn't clear in the Bugzilla. The bug was specifically for ClusterOperator status rather than ClusterVersion. (The logic here: https://github.com/openshift/console/blob/master/frontend/public/module/k8s/cluster-operator.ts#L18-L21)